pynest/examples/pynest-microcircuit/readme.md

```diff
@@ -0,0 +1,29 @@
+# Microcircuit Readme
+
+Authors: Hendrik Rothe, Hannah Bos, Sacha van Albada   
+May 2016
+
+## Description ##
+This is a PyNEST implementation of the SLI cortical microcircuit from the microcircuit model by Potjans and Diesmann (2014): The cell-type specific
```

Comment:
This is a PyNEST implementation of the microcircuit model ...
(no need to mention SLI)